core/src/main/java/com/klaytn/caver/crypto/KlayCredentials.java

```diff
@@ -63,6 +104,52 @@ public static KlayCredentials create(ECKeyPair ecKeyPair, String address) {
         return new KlayCredentials(ecKeyPair, address);
     }
 
+    public static KlayCredentials create(List<ECKeyPair> ecKeyPair, String address) {
+        return new KlayCredentials(ecKeyPair, null, null, address);
+    }
+
+    public static KlayCredentials create(List<ECKeyPair> ecKeyPairForTransaction, List<ECKeyPair> ecKeyPairForUpdate, List<ECKeyPair> ecKeyPairForFeePayer, String address) {
+        return new KlayCredentials(ecKeyPairForTransaction, ecKeyPairForUpdate, ecKeyPairForFeePayer, address);
+    }
+
+    public static KlayCredentials create(ECKeyPair[] ecKeyPairsArrayForTransaction, ECKeyPair[] ecKeyPairsArrayForUpdate, ECKeyPair[] ecKeyPairsArrayForFee, String address) {
+        List<ECKeyPair> ecKeyPairsForTransaction = new ArrayList<>(Arrays.asList(ecKeyPairsArrayForTransaction));
```

Comment:
Is this really necessary? `new ArrayList<>`?

Reply:
You're right. Confused the grammar.